api/nodes/views.py

```diff
@@ -293,11 +305,30 @@ def perform_create(self, serializer):
     # overrides BulkDestroyJSONAPIView
     def allow_bulk_destroy_resources(self, user, resource_list):
         """User must have admin permissions to delete nodes."""
+        if is_truthy(self.request.query_params.get('skip_uneditable', False)):
+            return any([node.has_permission(user, ADMIN) for node in resource_list])
         for node in resource_list:
```

Comment:
Unimportant, but in keeping with the `bulk_boolean_fn([list_comprehension])` above, the rest of this method could be simplified to 
```python
return all([node.has_permission(user, ADMIN) for node in resource_list])
```